src/index.spec.ts

```diff
@@ -28,17 +18,17 @@ describe('isEqual', function () {
     expect(isEqual(null, undefined)).to.equal(true);
     expect(isEqual(undefined, undefined)).to.equal(true);
     expect(isEqual(NaN, NaN)).to.equal(true);
-    expect(isEqual(null, NaN)).to.equal(true);
-    expect(isEqual(undefined, NaN)).to.equal(true);
+    expect(isEqual(null, NaN)).to.equal(false);
+    expect(isEqual(undefined, NaN)).to.equal(false);
```

Comment:
Why was this changed?

If intentional, the test should be moved out of the test block that says it tests for `true`.